[CS:GO] VMF basado en el RMF actualizado de 1.6.

WorstAquaPlayer opened the pull request:
**Texto del Pull Request cambiado desde el commit "VMF ahora basado en el RMF actual de 1.6."**

**Cambios:**
- El VMF ahora es uno el cual esta basado en el actualizado de 1.6:
   - Se hicieron cambios en las entidades "light_environment" y se añadieron las entidades "logic_auto" y "env_tonemap_controller" para ajustar la iluminación así es parecida a la versión de 1.6.
   - Se cambiaron las entidades "info_player_start" y "info_player_deathmatch" por "info_player_counterterrorist" y "info_player_terrorist".
   - Se añadieron las entidades "info_deathmatch_spawn" que estaban en el VMF original de este repositorio, aunque se modifico la posición de uno solo, el cual estaba posicionado en donde antes faltaba el brush cerca del spawn de Ts.
   - Se añadieron las entidades "func_buyzone" que estaban en el VMF original de este repositorio, aunque se modifico la formación uno, el **func_buyzone** de los terroristas se iba por debajo del mapa, causando un leak, así que se elevaron los vértices de abajo para mantener arriba su posición original.
- Añadido a la misma carpeta el BSP compilado y empacado con todas las texturas.
- Se elimino el archivo **.vmx**, solo es una copia previa del .vmf y no hace falta.
- Este VMF al ser directo del 1.6, utilice sus texturas original del WAD y las adapte al formato de Source. Esto lo hace incompatible con las texturas escaladas de 3kliksphilip, ya que las texturas están dimensionadas de maneras diferentes.
- Ya que las texturas originales del repositorio son incompatibles, fueron removidas.
- Se añadio el archivo **light.rad**, el cual puede ser utilizado por VRAD a la hora de compilar. La función de este archivo es que, escribiendo el nombre de una textura y los valores de luz, haga que una textura emita luz. Lo incluí porque la textura de luz utilizada en de_dust2 es una que proviene de Half-Life, y esta incluida en todos los compiladores, así que es probable que originalmente se haya utilizado esto.

**Capturas de este nuevo VMF:**
![scr1](https://user-images.githubusercontent.com/25833407/108050601-98d48780-7028-11eb-8da3-50971f94bbfc.png)

![scr3](https://user-images.githubusercontent.com/25833407/108050648-a853d080-7028-11eb-8f8d-c68cb7fd3e13.png)

![scr2](https://user-images.githubusercontent.com/25833407/108050659-adb11b00-7028-11eb-8d11-4bc85e5e30dc.png)

![scr4](https://user-images.githubusercontent.com/25833407/108050669-b144a200-7028-11eb-8ecb-1b9acc7ebf11.png)

**Para compilar el VMF modificado utilice el CompilePal, con el VRAD configurado de esta manera:**
![vrad_conf_new](https://user-images.githubusercontent.com/25833407/108051241-6b3c0e00-7029-11eb-85d4-b61eebc341e5.png)

WorstAquaPlayer committed:
Añadido nuevas texturas de Skybox

WorstAquaPlayer committed:
Cambio de iluminación y pequeño arreglo.

El BSP esta compilado del VMF de esta misma carpeta.
También se hizo un cambio en todas las texturas NULL, que ademas de estar sin empaquetar, es mejor utilizar nodraw en CS:GO.

WorstAquaPlayer committed:
Removidas las texturas de desarrollo

No es necesario incluir/empaquetar las texturas de la carpeta tool si estas vienen con el juego/editor.

rolivencia commented:
Varias de estas manqueadas pasaron dado que no me puse a reaprender cómo funciona todo para Source. Hago un chequeo y después te estoy dando una devolución.

WorstAquaPlayer committed:
Modificado el RMF de 1.6

WorstAquaPlayer committed:
Revert "Modificado el RMF de 1.6"

This reverts commit 89198fc38d592efc63fcf1d8736f94cfc17cbd1f.

WorstAquaPlayer commented:
Perdón por los 2 commits recientes, no tengo mucha experiencia en GitHub y al querer hacer otro pull he mezclado un commit, pero el ultimo lo revirtió.

rolivencia commented:
Asumiendo el trabajo hecho para el .RMF de 1.6, el cual es el origen para esta adaptación para CSGO, asumo que esta no será la versión final del mapa para CSGO, sino que seguirá sufriendo transformaciones para adaptarla.

De mi parte puedo aceptar el PR y dejar actualizado VMF y compilación y también podría subir el mapa al Workshop, pero por cuestiones de consistencia, tal vez sería mejor planificar una versión posterior y, ya que la herramienta existe, generar una checklist en un issue.

WorstAquaPlayer committed:
VMF ahora basado en el RMF actual de 1.6.

El VMF de CS:GO ahora esta basado en el RMF actualizado de 1.6.
Se elimino de_dust2_largo.vmx porque es tan solo una copia previa del VMF.
Todas las texturas del nuevo VMF están basadas en su versión original en WAD. Esto las hace incompatibles con las texturas escaladas de 3kliksphilip.
Ya que las texturas anteriores son incompatibles, fueron removidas del repositorio y fueron añadidas las que utiliza ahora el VMF.
Fue añadido light.rad, este archivo lo utiliza el VRAD, lo que hace este archivo es que las texturas cuyo nombre estén ahí emitan luz.

WorstAquaPlayer committed:
Añadido "func_buyzone" faltantes.

WorstAquaPlayer commented:
Modifique el texto entero del Pull Request porque al ser un VMF diferente no veo porque dejar los cambios anteriores.

> [...] tal vez sería mejor planificar una versión posterior [...]

Por mi parte creería que no habría mas cambios en el mapa, no se si tengas alguna idea para modificar/agregar u otra cosa, si ese es el caso entonces supongo que si, se debería hacer una issue para planificar los cambios.

WorstAquaPlayer committed:
Removido filtro pixelado en las texturas
